### PR TITLE
Code review first project

### PR DESCRIPTION
**Pull request by gagc056**

### CODE REVIEW

File: bubble_sort_by.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                    i=0
+                    if  yield(array[i] ,array[i+1])>0
+                        array[i], array[i+1] = array[i+1], array[i]
+                    end
+                    i+=1
+                end
+        else
+        return array
+        end
+    end
```
Comment (samgaco) [inline bubble_sort_by.rb]: This end is closing the function, which seems to be a mistake.

**Review by samgaco — CHANGES_REQUESTED** [review]
Hi @JuanPabloGil and @lhscar56 :high_brightness: 

bubble_sort_by seems to not be working properly, please address it and submit back for another code review.

-----------------------

moving on :arrow_down: 
* **[For your next code review requests]**  Add a short summary of changes in Pull Request description

---------------------------

* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Hi, Team, everything works now, however stickler has not been added to this project yet, can we make that work, and fix every known error before requesting for a review. You can find stickler details on the [pathwright path](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/57421588/) for this project

Submit for review, when this is done :pray:

**Review by BalintMendli — CHANGES_REQUESTED** [review]
Hi @lhscar56 @JuanPabloGil ,

Great job on the project. 👏 

However, Stickler is still not working. Make sure get the filename `.stickler.yml` right.

Also
- authorize Stickler to have access to the repo [here](https://github.com/settings/installations),
- enable repo on https://stickler-ci.com/,

if you haven't done so already.

When it is working fix any lint errors the code may have.

**Please submit another review request when it's done.**

**Review by bolah2009 — APPROVED** [review]
Linter seems to be working fine now. 
It's a :+1: from me.

#### :heavy_check_mark: You can merge to master.

-----

# Let's make this Pull Request look more professional.
  - [NEXT PROJECT]: Use meaningful commits messages. [Learn more here](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) and Check [this awesome article](https://chris.beams.io/posts/git-commit/)
  - [REQUIRED] Use meaningful names for  Pull Requests.
  - [REQUIRED] Add a short summary of changes in the Pull Request description.

# Rename PR
  rename PR
-----
# Update PR description
  update-PR-description-html

**Pull request merged**